src/test/groovy/jetbrains/buildServer/auth/oauth/OAuthClientTest.groovy

```diff
@@ -88,9 +88,9 @@ class OAuthClientTest extends Specification {
     def "should fetch user data"() {
         setup:
         def token = 'test_token'
-        server.enqueue(new MockResponse().setBody(JSONValue.toJSONString([id: 'user', name: 'userName', email: 'email'])))
+        server.enqueue(new MockResponse().setBody(JSONValue.toJSONString([id: 'user', name: 'userName', email: 'email', groups: ['dev-']])))
```

Comment:
could you create separate test for group and leave this test case for default case